Update asset-model.rst (#197)
digital-asset/daml-finance: docs/source/concepts/asset-model.rst

An ``owner`` must have an account contract with a ``custodian`` before a holding
contract can be created.

> This is similar to how, in the real world, you need to open a bank account

Reviewer: ```suggestion
As in the real world, you need to open a bank account
```
